app/login/page.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import Image from 'next/image';
+
+export default function LoginPage() {
+  return (
+    <div className="flex-1 flex">
+      <main className="flex-1 flex items-center justify-center p-7">
+        <div className="w-full max-w-md p-8 rounded-lg border">
+          <h2 className="text-2xl font-bold text-center mb-4">로그인</h2>
+          <p className="text-center text-gray-600 mb-8">
+            기존에 사용하는 계정으로 간편하게 로그인하세요.
+          </p>
+
+          <div className="space-y-4">
+            <button className="w-full flex items-center justify-center gap-2 bg-[#FEE500] text-black py-3 px-4 rounded-md">
```

Comment:
```suggestion
            <button className="w-full flex items-center justify-center gap-2 bg-[#FDDC3F] text-black py-3 px-4 rounded-md">
```

색깔이 안맞는거 같아여!! 피그마로 색 슬쩍 갖고왔슴니다